cli/cmd/install.go

```diff
@@ -420,6 +421,12 @@ func (options *installOptions) validateAndBuild(stage string, flags *pflag.FlagS
 	values.Global.IdentityTrustDomain = identityValues.TrustDomain
 	values.Stage = stage
 
+	// Update Configuration of Add-ons from config file
```

Comment:
Can we leave this and the `--config` flag out for now? I understand we need a way to override addon configuration without bloating the CLI flags. I want to better understand this is an addon-only flag, or will it become something more general like Helm's `-f` option. I am also a bit hesitant about the fork repo.

For now, I think we can just use the addon defaults defined in the `values.yaml`, right?

Reply:
Yep, Right Now, as per the code too, we only overwrite the config of add-ons, As users would want a way to toggle the add-on on/off. Without the config flag, Toggle would require each add-on having its own flag which can add unnecessary work.

And also users would definitely want a way to overwrite fields like resouce limits, values, etc without having to change templates.

The next work, would be to make the add-on cm have linkerd2 values too. 

Yep, I did not like forking the repo, but thats the only solution I came across. :\

Reply:
I can definitely try something here, if its a problem.

Reply:
> Without the config flag, Toggle would require each add-on having its own flag which can add unnecessary work.

Just use Helm to install addon for now. I picture we will tackle the config PR immediately after this PR.

Basically, there are 3 things I want to know:

1. Will this flag be used only for addon configs? If yes, its name should reflect that.
1. How does a user know what is the supported content of this file? Technically, they can look up the Linkerd's chart `values.yaml` in the repo. Even then, how do they know which of those variables are supported? Can we do better?
1. The logic used to merge this file with the addons' defaults defined in `values.yaml` should be unit tested.

Reply:
> Yep, I did not like forking the repo, but thats the only solution I came across

Can you take a look to see if [`chartutil.Values.MergeInto()`](https://pkg.go.dev/k8s.io/helm@v2.16.3+incompatible/pkg/chartutil?tab=doc#Values.MergeInto) will do what you want?

Reply:
Yep, It works.
Thanks :)